devito/operator/operator.py

```diff
@@ -898,10 +899,14 @@ def _emit_apply_profiling(self, args):
         # thing that will be emitted
         for k, v in summary.items():
             rank = "[rank%d]" % k.rank if k.rank is not None else ""
-            oi = "OI=%.2f" % fround(v.oi)
-            gflopss = "%.2f GFlops/s" % fround(v.gflopss)
-            gpointss = "%.2f GPts/s" % fround(v.gpointss) if v.gpointss else None
-            metrics = ", ".join(i for i in [oi, gflopss, gpointss] if i is not None)
+            if v.gflopss:
```

Comment:
When does this happen?

Reply:
data streaming sections which only trigger data transfers (so they take up time) but no compute